addons/website_sale/views/templates.xml

```diff
@@ -1108,7 +1108,7 @@
                                         <div id="product_option_block" class="d-flex flex-wrap w-100"/>
                                     </div>
                                     <div id="contact_us_wrapper"
-                                         t-attf-class="{{'d-flex' if combination_info['prevent_zero_price_sale'] else 'd-none'}} oe_structure oe_structure_solo #{_div_classes}">
+                                         t-attf-class="{{'d-flex' if combination_info['prevent_zero_price_sale'] else 'd-none'}} oe_structure oe_structure_solo #{_div_classes} mb-2">
```

Comment:
it would be better to give mt on bellow block instead mb on above block this way we could avoid empty bottom space when bellow block is not visible